documentation/aks-upgrade.md

```diff
@@ -112,6 +112,9 @@ az aks nodepool get-upgrades --resource-group <ResourceGroup> --cluster-name <Cl
 1. Update the kubectl client version to match the AKS (Azure Kubernetes Service) cluster version in GitHub Actions.
     - You can achieve this using the set-kubectl action from the https://github.com/DFE-Digital/github-actions/set-kubectl
 
+1. After a successful upgrade, check metrics are still displaying and working well in Grafana
```

Comment:
Monitoring should be checked for each environment. This step should be before "Follow the same manual process" and show the different environment grafana URLs